compiler-rt/lib/sanitizer_common/sanitizer_libc.cpp

```diff
@@ -49,6 +49,16 @@ int internal_memcmp(const void* s1, const void* s2, uptr n) {
   return 0;
 }
 
+void *internal_memccpy(void *dest, const void *src, int c, uptr n) {
```

Comment:
It would be good to have a test for this. Unfortunately, I don't see any internal_* functions being tested in compiler-rt/test right now, but it would make sense to start with this one.